mmdet/core/visualization/image.py

```diff
@@ -505,20 +511,49 @@ def imshow_gt_det_bboxes(img,
         labels = np.array([id % INSTANCE_OFFSET for id in ids], dtype=np.int64)
         segms = (pan_results[None] == ids[:, None, None])
 
-    img = imshow_det_bboxes(
-        img,
-        bboxes,
-        labels,
-        segms=segms,
-        class_names=class_names,
-        score_thr=score_thr,
-        bbox_color=det_bbox_color,
-        text_color=det_text_color,
-        mask_color=det_mask_color,
-        thickness=thickness,
-        font_size=font_size,
-        win_name=win_name,
-        show=show,
-        wait_time=wait_time,
-        out_file=out_file)
+    if overlay_gt_pred:
+        img = imshow_det_bboxes(
+            img_with_gt,
+            bboxes,
+            labels,
+            segms=segms,
+            class_names=class_names,
+            score_thr=score_thr,
+            bbox_color=det_bbox_color,
+            text_color=det_text_color,
+            mask_color=det_mask_color,
+            thickness=thickness,
+            font_size=font_size,
+            win_name=win_name,
+            show=show,
+            wait_time=wait_time,
+            out_file=out_file)
+    else:
+        img_with_det = imshow_det_bboxes(
+            img,
+            bboxes,
+            labels,
+            segms=segms,
+            class_names=class_names,
+            score_thr=score_thr,
+            bbox_color=det_bbox_color,
+            text_color=det_text_color,
+            mask_color=det_mask_color,
+            thickness=thickness,
+            font_size=font_size,
+            win_name=win_name,
+            show=False)
+        img = np.concatenate([img_with_gt, img_with_det], axis=0)
+
+        plt.imshow(img)
```

Comment:
why the show logic is only allowed when overlay_gt_pred=False? show should work with both cases.

Reply:
when overlay_gt_pred=True, the show logic is in `imshow_det_bboxes`, https://github.com/open-mmlab/mmdetection/blob/240d7a31c745578aa8c4df54c3074ce78b690c34/mmdet/core/visualization/image.py#L358